lms/templates/dashboard.html

```diff
@@ -25,7 +26,13 @@
 <%
   cert_name_short = settings.CERT_NAME_SHORT
   cert_name_long = settings.CERT_NAME_LONG
-  catalog_link = settings.MARKETING_SITE_BASE_URL
+  uai_course_key_formats = getattr(settings, "UAI_COURSE_KEY_FORMATS", [])
+  request_path = request.get_full_path().lower()
+
+  if any(course_key in request_path for course_key in uai_course_key_formats):
+    catalog_link = getattr(settings, "MIT_LEARN_BASE_URL", getattr(settings, "MARKETING_SITE_BASE_URL", "#"))
+  else:
+    catalog_link = getattr(settings, "MARKETING_SITE_BASE_URL", "#")
```

Comment:
These changes are incorrect, dashboard URL is /dashboard, How can it ever match the course key formats? Also, the dashboard is disabled and redirects to the mitxonline dashboard via the redirects configured in django admin app `Redirects`. We can't really do anything here.

We should discuss this with @pdpinch to redirect /dashboard to either mitxonline or learn, we can't do it dynamically with current set of configuration/changes.